mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/task/SolveMailboxInconsistenciesTaskAdditionalInformationDTO.java

```diff
@@ -0,0 +1,115 @@
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ***************************************************************/
+
+package org.apache.james.mailbox.cassandra.mail.task;
+
+import java.time.Instant;
+
+import org.apache.james.json.DTOModule;
+import org.apache.james.server.task.json.dto.AdditionalInformationDTO;
+import org.apache.james.server.task.json.dto.AdditionalInformationDTOModule;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableList;
+
+public class SolveMailboxInconsistenciesTaskAdditionalInformationDTO implements AdditionalInformationDTO {
+    private static SolveMailboxInconsistenciesTaskAdditionalInformationDTO fromDomainObject(SolveMailboxInconsistenciesTask.Details details, String type) {
+        return new SolveMailboxInconsistenciesTaskAdditionalInformationDTO(
+            type,
+            details.getProcessedMailboxEntries(),
+            details.getProcessedMailboxPathEntries(),
+            details.getFixedInconsistencies(),
+            details.getConflictingEntries(),
+            details.getErrors(),
+            details.timestamp());
+    }
+
+    public static final AdditionalInformationDTOModule<SolveMailboxInconsistenciesTask.Details, SolveMailboxInconsistenciesTaskAdditionalInformationDTO> MODULE =
+        DTOModule
+            .forDomainObject(SolveMailboxInconsistenciesTask.Details.class)
+            .convertToDTO(SolveMailboxInconsistenciesTaskAdditionalInformationDTO.class)
+            .toDomainObjectConverter(SolveMailboxInconsistenciesTaskAdditionalInformationDTO::toDomainObject)
+            .toDTOConverter(SolveMailboxInconsistenciesTaskAdditionalInformationDTO::fromDomainObject)
+            .typeName(SolveMailboxInconsistenciesTask.SOLVE_MAILBOX_INCONSISTENCIES.asString())
+            .withFactory(AdditionalInformationDTOModule::new);
+
+    private final String type;
+    private final long processedMailboxEntries;
+    private final long processedMailboxPathEntries;
+    private final long fixedInconsistencies;
+    private final ImmutableList<ConflictingEntry> conflictingEntries;
+    private final long errors;
+    private final Instant timestamp;
+
+    public SolveMailboxInconsistenciesTaskAdditionalInformationDTO(@JsonProperty("type") String type,
+                                                                   @JsonProperty("processedMailboxEntries") long processedMailboxEntries,
+                                                                   @JsonProperty("processedMailboxPathEntries") long processedMailboxPathEntries,
+                                                                   @JsonProperty("fixedInconsistencies") long fixedInconsistencies,
+                                                                   @JsonProperty("conflictingEntries") ImmutableList<ConflictingEntry> conflictingEntries,
```

Comment:
`List`

Reply:
Nope, I prefer keeping the information that this is immutable.